All Core Devs Meetings/Meeting 58.md

```diff
@@ -0,0 +1,298 @@
+# Ethereum Core Devs Meeting 58 Notes
+### Meeting Date/Time: Friday, March 29, 2019 14:00 UTC
+### Meeting Duration:  1 hours 45 minutes
+### [GitHub Agenda Page](https://github.com/ethereum/pm/issues/89)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=v8Psbo8zY4Y)
+### Moderator: Hudson Jameson
+
+# Summary
+
+### DECISIONS MADE
+
+**DECISION 58.1** The [ProgPoW Carbon Vote](http://www.progpowcarbonvote.com/) signal will be shutting down in 13 days from today. Everyone that wishes to vote will need to do so before this date and ensure they leave their ETH in the address they voted from until after the specified block number.
+
+**DECISION 58.2** Going forward clients can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+**DECISION 58.3** Going forward Reseach can provide an update in the comments in the agenda. If there are any questions or anything specific to discuss a space will be left to do so.
+
+### ACTIONS REQUIRED
+
+**ACTION 58.1** Cat Herders to look at updating EIP1.
+
+**ACTION 58.2** Review the proposed solutions for Roadmaps in the [Ethereum Magicians](https://ethereum-magicians.org/t/more-frequent-smaller-hardforks-vs-less-frequent-larger-ones/2929) forum to decide if going forward the Core Devs adopts smaller hardforks rather than larger hardforks.
+
+**ACTION 58.3** Vitalik to format the currently proposed EIP-1559 so that it is correctly presented.
```

Comment:
Perhaps link it? https://github.com/ethereum/EIPs/issues/1559

Reply:
Updated.